util/subtree_linux.go

```diff
@@ -79,37 +80,35 @@ func listProc() ([]int, error) {
 			if err != nil {
 				continue
 			}
+
+			if pid == myPid {
+				continue
+			}
+
+			if !processHasMarkers(pid, key, logger) && !executableInWorkingDir(pid, workingDir, logger) {
```

Comment:
Same question about the working dir if it's the empty string.

Reply:
caught the empty string case